src/biotite/structure/io/mol2/file.py

```diff
@@ -0,0 +1,990 @@
+            1  UNL        C         2.835   -1.164    0.071
+            1  UNL        C         2.156   -2.382    0.096
+            1  UNL        C         2.879   -3.572    0.063
+            1  UNL        C         4.282   -3.558    0.021
+            1  UNL        C         4.974   -2.321   -0.013
+            1  UNL        C         6.385   -2.347   -0.040
+            1  UNL        C         7.089   -3.553   -0.030
+            1  UNL        C         6.399   -4.759   -0.002
+            1  UNL        C         5.004   -4.762    0.020
+            1  UNL        C         4.225   -1.121   -0.006
+            1  UNL        O         4.909    0.073   -0.087
+            1  UNL        C         4.218    1.277    0.018
+            1  UNL        C         2.884    1.328    0.147
+            1  UNL        N         5.001    2.423   -0.069
+            1  UNL        C         4.820    3.609    0.790
+            1  UNL        C         4.974    4.886   -0.048
+            1  UNL        O         6.238    4.918   -0.733
+            1  UNL        C         6.369    3.772   -1.593
+            1  UNL        C         6.282    2.457   -0.799
+     
+
+
+
+    """
+
+
+    sybyl_to_biotite_bonds = {
+
+        "1": struc.BondType.SINGLE,
+        "2": struc.BondType.DOUBLE,
+        "3": struc.BondType.TRIPLE,
+        "am": None,             # amide not really supported in biotite yet
+        "ar": struc.BondType.AROMATIC_SINGLE,  # questionable if this is okay since we have up to 3 formats for aromatic bonds
+        "du": None,             # no idea what dummy is
+        "un": struc.BondType.ANY,
+        "nc": None,
+    }
+    biotite_bonds_to_sybyl = {
+        1: "1",
+        2: "2",           
+        3: "3",
+        5: "ar",
+        0: "un",                    
+    }
+
+    supported_charge_types = [
+        "NO_CHARGES", "DEL_RE", "GASTEIGER", 
+        "GAST_HUCK", "HUCKEL", "PULLMAN", 
+        "GAUSS80_CHARGES", "AMPAC_CHARGES", 
+        "MULLIKEN_CHARGES", "DICT_CHARGES", 
+        "MMFF94_CHARGES", "USER_CHARGES"
+    ]
+        
+        
+    supported_mol_types = [
+        "SMALL", "BIOPOLYMER", "PROTEIN", "NUCLEIC_ACID", "SACCHARIDE"
+    ]    
+        
+    sybyl_status_bit_types = [
+        "system", "invalid_charges", "analyzed", "substituted", "altered", "ref_angle"
+    ]
+
+            
+    # field that contains all accepted two letter elements
+    # this is used for deriving if sperate atom_name and element 
+    # columns should be generated. If only two letter atom_name entries in the 
+    # mol2 file are found that are contained here, the atom_name entry will be
+    # assumed to only contain element names.        
+    elements_twoLetters = [
+        "LI", "NA", "MG", "AL",
+        "SI", "CA", "CR", "MN",
+        "FE", "CO", "CU", "CL",
+        "BR", "ZN", "SE", "MO",
+        "SN", "AR"
+    ]    
+    
+    @staticmethod        
+    def get_sybyl_atom_type(atom, bonds, atom_id):
```

Comment:
Is this function meant to be private?

For performance reasons, I would suggest to give the function all bonds, element, and the atom index directly. The bonds for each atom can be more efficiently obtained via `get_all_bonds()`.

Reply:
As discussed elsewher this should remain a static function. I renamed it to
**transcribe_sybyl_atom_type** to make clear that this is not a getter function